src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileMetadataRouteSpec.scala

```diff
@@ -100,6 +101,17 @@ class FileMetadataRouteSpec extends AnyFlatSpec with Matchers with TestRequest w
     checkNoFileMetadataAdded()
   }
 
+  "addFileMetadata" should "add the checksum validation result if this is a checksum update" in {
+    createClientFileMetadata(defaultFileId)
+    runTestMutation("mutation_alldata", validBackendChecksToken("checksum"))
+    checkValidationResultExists(defaultFileId)
+  }
+
+  "addFileMetadata" should "add the checksum validation result if this is not a checksum update" in {
```

Comment:
Is there a "not" missing from the "add the checksum validation result"?

Reply:
The copy and paste demon strikes again. I've updated it, cheers.